R/row_to_names.R

```diff
@@ -40,3 +67,64 @@ row_to_names <- function(dat, row_number, remove_row = TRUE, remove_rows_above =
     dat
   }
 }
+
+#' Find the header row in a data.frame
+#' 
+#' @details
+#' If \code{...} is missing, then the first row with no missing values is used.
+#' If \code{...} has a single character argument, then the first column is
+#' searched for that value.  If \code{...} has a named numeric argument, then
+#' the value of the argument is searched for the name (see the examples).  If
+#' more than one row is found matching a value that is searched for, the first
+#' matching row will be returned (with a warning).
+#' 
```

Comment:
Maybe we note that when searching for a string, the complete row requirement no longer applies.  We could add in a new short paragraph here:

`When searching for a specified column name, the first row with a match will be returned, regardless of the completeness of the rest of that row.`

Is that an improvement, or just more wordy and confusing?  It might be able to replace the last sentence in the preceding paragraph about what happens with multiple matches.

Reply:
That's a good clarification.  I added it with a minor modification.